src/Avalonia.Themes.Fluent/Controls/SplitButton.xaml

```diff
@@ -0,0 +1,247 @@
+    <converters:MarginMultiplierConverter x:Key="SeparatorBorderMultiplier" Top="True" Bottom="True" Indent="1" />
+  </Styles.Resources>
+
+  <Style Selector="SplitButton">
+    <Setter Property="Background" Value="{DynamicResource SplitButtonBackground}" />
+    <Setter Property="Foreground" Value="{DynamicResource SplitButtonForeground}" />
+    <Setter Property="BorderBrush" Value="{DynamicResource SplitButtonBorderBrush}" />
+    <Setter Property="BorderThickness" Value="{DynamicResource SplitButtonBorderThemeThickness}" />
+    <Setter Property="HorizontalAlignment" Value="Left" />
+    <Setter Property="VerticalAlignment" Value="Center" />
+    <!--<Setter Property="UseSystemFocusVisuals" Value="True" />
+    <Setter Property="FocusVisualMargin" Value="-3" />-->
+    <Setter Property="KeyboardNavigation.IsTabStop" Value="True" />
+    <Setter Property="Focusable" Value="True" />
+    <Setter Property="Padding" Value="{DynamicResource ButtonPadding}" />
+    <Setter Property="CornerRadius" Value="{DynamicResource ControlCornerRadius}" />
+    <Setter Property="Template">
+      <Setter.Value>
+        <ControlTemplate>
+          <Grid>
+            <Grid.ColumnDefinitions>
+              <ColumnDefinition Width="*" />
+              <ColumnDefinition Width="Auto" />
+              <ColumnDefinition Width="Auto" />
+            </Grid.ColumnDefinitions>
+
+            <Button x:Name="PART_PrimaryButton"
+                    Grid.Column="0"
+                    MinWidth="{DynamicResource SplitButtonPrimaryButtonSize}"
+                    Foreground="{TemplateBinding Foreground}"
+                    Background="{TemplateBinding Background}"
+                    BorderThickness="{TemplateBinding BorderThickness, Converter={StaticResource PrimaryButtonBorderMultiplier}}"
+                    BorderBrush="{TemplateBinding BorderBrush}"
+                    Content="{TemplateBinding Content}"
+                    ContentTemplate="{TemplateBinding ContentTemplate}"
+                    Command="{TemplateBinding Command}"
+                    CommandParameter="{TemplateBinding CommandParameter}"
+                    FontFamily="{TemplateBinding FontFamily}"
+                    FontSize="{TemplateBinding FontSize}"
+                    FontWeight="{TemplateBinding FontWeight}"
+                    HorizontalAlignment="Stretch"
+                    VerticalAlignment="Stretch"
+                    HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
+                    VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
+                    Padding="{TemplateBinding Padding}"
+                    Focusable="False"
+                    KeyboardNavigation.IsTabStop="False" />
+
+            <Border x:Name="SeparatorBorder"
+                    Grid.Column="1"
+                    Background="Transparent"
+                    Width="{DynamicResource SplitButtonSeparatorWidth}"
```

Comment:
I added a new SplitButtonSeparatorWidth resource. This allows easy customization and will be needed in Fluent v2 where the gap is removed and replaced with just a border. This also allows easier app customization.

Note that the term 'width' is used since this can only be width. Traditionally 'Size' is used where it may be either height/width.